tests/cypress/integration/directives/x-on.spec.js

```diff
@@ -770,11 +770,11 @@ test(
         get("@button").trigger("click", {});
         get("@alt").should(notBeChecked());
         get("@button").trigger("click", { ctrlKey: true, shiftKey: true, metaKey: true, altKey: true });
-        get("@shift").as("shift").should(beChecked());
-        get("@ctrl").as("ctrl").should(beChecked());
-        get("@meta").as("meta").should(beChecked());
-        get("@alt").as("alt").should(beChecked());
-        get("@cmd").as("cmd").should(beChecked());
+        get("@shift").should(beChecked());
+        get("@ctrl").should(beChecked());
+        get("@meta").should(beChecked());
+        get("@alt").should(beChecked());
+        get("@cmd").should(beChecked());
     }
```

Comment:
Not sure why this wasn't causing an issue before (or maybe it was and it was hidden), but in the latest cypress this cypress this throws a max stack depth error (as you would expect it to I think?)

Reply:
funny.

I think that's my mistake. I added those in one spot, and maybe copy pasted it without care and it happened to work on the old version.